x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -378,7 +411,7 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
               )
             }
             aria-label={i18n.translate('xpack.lens.indexPattern.terms.orderBy', {
-              defaultMessage: 'Order by',
+              defaultMessage: 'Rank by',
```

Comment:
Is this a ranking system? Or an order system?

Reply:
I feel it's a tricky question :D .
I'd say it is a `ranking system` here to differentiate from `order` which sounds more as a postprocessing task in this context.